ci-operator/jobs/openshift/kueue-operator/openshift-kueue-operator-main-periodics.yaml

```diff
@@ -301,3 +301,68 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```

Comment:
I think you have to add a jira hook so we will be notified on failures.

/hold

Reply:
https://github.com/openshift/release/blob/5c5757244ed6777e130518e684477d04a66c32f9/ci-operator/jobs/openshift/kueue-operator/openshift-kueue-operator-main-periodics.yaml#L14

Reply:
Fixed